praw/models/reddit/mixins/__init__.py

```diff
@@ -118,12 +118,17 @@ def lock(self):
             API_PATH["lock"], data={"id": self.thing.fullname}
         )
 
-    def remove(self, spam=False):
+    def remove(self, mod_note="", reason_id=None, spam=False):
```

Comment:
Thanks for sorting these. In this particular case, I don't think we should move the position of `spam` since people may be using `remove(True)`, and this would result in a a break. In a future where we use python 3.8+ then we can require that people use keyword arguments, but at this point a change in position of public method arguments is a breaking change.

Reply:
Yeah I wondered about that. I reverted to: `spam`, `mod_note`, `reason_id`